crypten/mpc/primitives/beaver.py

```diff
@@ -35,12 +36,33 @@ def __beaver_protocol(op, x, y, *args, **kwargs):
         x.size(), y.size(), op, device=x.device, *args, **kwargs
     )
 
-    # Vectorized reveal to reduce rounds of communication
     from .arithmetic import ArithmeticSharedTensor
 
-    eps_del = ArithmeticSharedTensor.reveal_batch([x - a, y - b])
-    epsilon = eps_del[0]
-    delta = eps_del[1]
+    if crypten.mpc.config.security == "active":
+        f, g, h = provider.generate_additive_triple(
+            x.size(), y.size(), op, device=x.device, *args, **kwargs
+        )
+
+        # Generate an MPCTensor from random shares and when reconstructing it
+        # it would be a random value
+        t_share = generate_random_ring_element(
+            a.size(),
+            generator=comm.get().get_generator(0, device=x.device),
+            device=x.device
+        )
+        t_mpc = ArithmeticSharedTensor.from_shares(share=t_share, precision=0)
```

Comment:
Note to self: this functionality seems to be useful in a lot of places (especially when generating MAC shares). I should implement a PRSS function to do this. Maybe add a TODO comment here to implement PRSS.

Reply:
I could also move this functionality to a ```@staticmethod``` in ```ArithmeticSharedTensor``` (as there is ```PRZS``` only that it would be called ```PRSS```). If you plan to put it somewhere else I could leave it here?

Reply:
Do you think we should initialize a ```generator``` specifically designed for this (I am talking about line 50). For the moment we use the generator with index 0 (but that one is also used for PRZS)

Reply:
We cannot use the same generators as used by PRZS as those are shared with another party (and in the two party case, both generators are shared by both parties, making those generators trivially insecure). This is okay for PRZS since there is no information contained in the PRZS shares.

Reply:
Regarding your first comment, that is my thought exactly.

Reply:
I can push in an hour a commit to add the ```PRSS``` to the ```ArithmeticSharedTensor```. Or do you already have the commit prepared?